packages/material-ui-unstyled/src/SliderUnstyled/SliderUnstyled.js

```diff
@@ -10,6 +10,7 @@ import {
   unstable_useForkRef as useForkRef,
   unstable_useControlled as useControlled,
 } from '@material-ui/utils';
+import { visuallyHidden } from '@material-ui/system';
```

Comment:
This one is tough. We have been talking about making the system includes the Box and depend on @material-ui/styled-engine, to make it truly useful on its own. if we don't, unstyled should probably be in the core package directly.

So, assuming we move forward in this direction, it would make unstyled depend on emotion, which I think is a deal-breaker for the purpose of the unstyled package.

On the other hand, if we move the `visuallyHidden` to the styled version, we would need to handle it for the unstyled version.

I also want to raise #19938 where we talk about the CSP implications of using inline-style.

cc @mnajdova

Reply:
Definitely a hard one.. I see two options. The first one to use inline style, the second one is to bundle some CSS module that developers can import and use, that will have the necessary styles.. Both have their pros/cons. I am inclined to say maybe exporting CSS modules is not a bad path forward, but until we have that set up, I'd as we could go with inline styles... I am open to other ideas if there are any..

Reply:
- So you are happy to ignore the CSP issue, we ignore it for simplicity, OK, why not, it what we have done so far.
- How do we resolve the package dependency issue with inline-style?

Reply:
> So you are happy to ignore the CSP issue, we ignore it for simplicity, OK, why not, it what we have done so far.

For now, we are not doing anything worse, but will be indicator that maybe we should move to CSS modules.

> How do we resolve the package dependency issue with inline-style?

I don’t think we should have dependency to the system, we can duplicate the styles necessary.

Reply:
> we can duplicate the styles necessary.

Maybe move `visuallyHidden` into @material-ui/utils?

Reply:
> For now, we are not doing anything worse, but will be indicator that maybe we should move to CSS modules.

I kind of feel that we will need inline-style anyway for dynamic use cases, so the CSP thing might be a lost cause anyway.

Reply:
Agree, let’s overcome that issue once we have it.. I don’t see the CSP as the biggest issue, but rather the style overrides. For the slider, which changes the styles dynamically based on dragging, inline style is the only solution anyway

Reply:
> Ok, maybe we can move visuallyHidden into @material-ui/utils and keep using inline style? 

Sounds good to me 👍

Reply:
Will do!

Reply:
Do I need to do anything special after moving this, like updating the CHANGELOG so it gets put in release notes that it has moved? Or will that be handled separately?

Reply:
Also looking in the utils file some things are `.ts` files, is it ok to convert it to be a typescript file, or should I just make a `.d.ts` file along side it since things are re-exported from a `index.ts` files instead of an `index.js` file like in system?

Reply:
I don't think that we need to update the Changelog as we can make it non breaking by keeping system to export it. If there are places in the documentation where it's used, we might want to update them to point to utils. You can migrate it to TypeScript, we try to write new code in TypeScript.